pkg/blobinfocache/boltdb.go

```diff
@@ -220,6 +220,7 @@ func (bdc *boltDBCache) RecordDigestUncompressedPair(anyDigest digest.Digest, un
 // RecordKnownLocation records that a blob with the specified digest exists within the specified (transport, scope) scope,
 // and can be reused given the opaque location data.
 func (bdc *boltDBCache) RecordKnownLocation(transport types.ImageTransport, scope types.BICTransportScope, blobDigest digest.Digest, location types.BICLocationReference) {
+	scope.Opaque = "all"
```

Comment:
(If this should happen, it should happen inside the `docker` transport directly, by not setting a per-domain value, instead of stomping over the set value here in the transport-agnostic code.)